apps/emqx_modules/src/emqx_telemetry.erl

```diff
@@ -452,6 +453,36 @@ advanced_mqtt_features() ->
     AdvancedFeatures = emqx_modules:get_advanced_mqtt_features_in_use(),
     maps:map(fun(_K, V) -> bool2int(V) end, AdvancedFeatures).
 
+get_authn_authz_info() ->
```

Comment:
Implement this function in two parts 
authn part in emqx_authn app
authz part in emqx_authz app
And this wrapping function should not have to assume their internal data structure